src/storm-pomdp/modelchecker/BeliefExplorationPomdpModelCheckerOptions.h

```diff
@@ -36,9 +46,18 @@ namespace storm {
                 // Controls which observations are refined.
                 ValueType obsThresholdInit = storm::utility::convertNumber<ValueType>(0.1);
                 ValueType obsThresholdIncrementFactor = storm::utility::convertNumber<ValueType>(0.1);
-                
+
+                uint64_t clippingGridRes = 2;
+
+                bool skipHeuristicSchedulers = false;
+
                 ValueType numericPrecision = storm::NumberTraits<ValueType>::IsExact ? storm::utility::zero<ValueType>() : storm::utility::convertNumber<ValueType>(1e-9); /// Used to decide whether two beliefs are equal
                 bool dynamicTriangulation = true; // Sets whether the triangulation is done in a dynamic way (yielding more precise triangulations)
+
+                storm::builder::ExplorationHeuristic explorationHeuristic = storm::builder::ExplorationHeuristic::BreadthFirst;
+
+                storm::solver::MinMaxMethod preProcMinMaxMethod = storm::solver::MinMaxMethod::SoundValueIteration;
```

Comment:
I suggest to have a `storm::Environment preProcEnvironment` instead if this really needs to be configured "from the outside"